src/Models/restaurants.json

```diff
@@ -0,0 +1,141 @@
+[  
+    {
+        "restaurantName": "MBM Trinbago Meals",
+        "address": "Pool Road, Birmingham B66 3DB",
```

Comment:
Can I be really annoying and ask for the address to be an object? Will help eventually when it needs to be stored in the database, for any location stuff you wanna do in the future and even displaying the address on the frontend.
 E.g. 
```
"address": {
  "firstLine": "Pool Road",
  "secondLine": null,
  "city": "Birmingham",
  "postCode": "B66 3DB"
}
```